### PR TITLE
Nailgun NTP improvements

### PR DESCRIPTION
**Pull request by mattymo**

### CODE REVIEW

File: deployment/puppet/cobbler/templates/dhcp-enter-hooks.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (aglarendil) [inline deployment/puppet/cobbler/templates/dhcp-enter-hooks.erb:1]: how is this script related to ntp improvements?

Comment (mattymo) [inline deployment/puppet/cobbler/templates/dhcp-enter-hooks.erb:1]: We need dns resolution available during first boot of master node for Fuel Menu and for NTP. It fixes the issues related to time jumps in deployment, which makes following logs difficult. We will enable resolution at boot, then puppet will block dhclient from updating /etc/resolv.conf for DHCP config and set /etc/resolv.conf manually

**Comment by teran** [comment]
Verified, works fine.

**Comment by aglarendil** [comment]
+1
